### PR TITLE
使用 typedoc 生成 api 文档

### PR DESCRIPTION
**Pull request by simaQ**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] All tests pass and/or benchmarks are included
- [ ] Commit message follows commit guidelines
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "lib": "run-p lib:*",
     "build": "run-s clean lib",
     "ci": "run-s build coverage",
-    "coveralls": "cat ./coverage/lcov.info | coveralls"
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "doc": "rm -rf docs/api && typedoc"
```
Comment (hustcc) [inline package.json]: 用 rimraf，不然可能在 window 系统中找不到 rm

Comment (simaQ) [reply package.json]: 好捏，done

**Comment by simaQ** [comment]
相关的代码注释规范：https://www.yuque.com/antv/g2-docs/cchwwx

**Comment by simaQ** [comment]
相对来说 type 的注释比较简单的，只要做到两点：
1. 类型定义充分
2. 提供变量描述即可

可以在 code 编辑器中安装 Comments in Typescript 插件，提高效率

**Pull request merged**